docs/product-user-guide/src/main/asciidoc/guided-decision-tables-columns-create-proc.adoc

```diff
@@ -1,25 +1,29 @@
 [id='guided-decision-tables-columns-create-proc']
 = Adding columns to guided decision tables
 
-After you have created the guided decision table, you can define and add various types of columns within the decision table editor.
+After you have created the guided decision table, you can define and add various types of columns within the decision table designer.
 
 .Prerequisite
-All column-related data objects, such as Facts and Fields, have been created within the same package where the guided decision table is found, or have been imported from another package in *Data Objects* -> *New item* of the guided decision table editor. These objects and assets will be required for some of the parameters for adding columns to a guided decision table. For descriptions of these parameters, see the "Required Column Parameters" segments for each column type in <<guided-decision-tables-columns-types-con>>. For details about creating data objects, see **(expand content from data-objects-create-proc.adoc).
+All column-related data objects, such as Facts and Fields, have been created within the same package where the guided decision table is found, or have been imported from another package in *Data Objects* -> *New item* of the guided decision table designer. These objects and assets will be required for some of the parameters for adding columns to a guided decision table.
```

Comment:
I don't understand this paragraph and I know how it should work!

I know what you're trying to explain, but I don't think the wording is very clear.

Reply:
Haha! Well that's a problem then. This any better? My best shot: http://file.rdu.redhat.com/~sterobin/BXMSDOC-2008/#guided-decision-tables-columns-create-proc